### ISSUE TITLE
Move deployment repo to bcgov group

### ISSUE THREAD

**Issue by CraigClark** [bcgov/MFIN-Data-Catalogue#186]
Currently the repo we use for deployments is in the _bcgov-c_ group on gitHub. It has to be moved to the _bcgov_ group to comply with BC policy.

The repo has been vetted for privacy and has been public for a while now. 

## Task description

1. move the [devOps](https://github.com/bcgov-c/tenant-gitops-ea352d) repo from the [staging group](https://github.com/bcgov-c) to the [public group](https://github.com/bcgov).
2. grant users `sylus` and `smulvih2` admin permissions 
3. grant users `CraigClark` and `lkmorlan` write permissions
4. configure Argo to point to the new location for `tenant-gitops-ea352d`

@sylus FYI

**Comment by NicoledeGreef** [comment]
Started this process; suggested it be a new DevOps Request Type: https://github.com/BCDevOps/devops-requests/issues/1557
Actual request has not yet been submitted; waiting for response on the above request type suggestion as it may guide the request in a different direction.

**Comment by NicoledeGreef** [comment]
@CraigClark @sylus @lkmorlan 
Please see comments in https://github.com/BCDevOps/devops-requests/issues/1557

**Comment by CraigClark** [comment]
@NicoledeGreef Looking for confirmation, it looks like we don't need to worry about moving it? 

If I understand correctly, the repo should not be public because of the github actions? Right now, even though it is in bcgov-c it is public, anyone can see the [actions on this repo](https://github.com/bcgov-c/tenant-gitops-ea352d/actions). Should it be private?

**Comment by NicoledeGreef** [comment]
@CraigClark I think we vetted the content before making it "public" so it's not red hot.  I can peruse the contents if I'm not logged into GH. We might as well change it back to private if it's not leaving bcgov-c.

With the new input from Platform Services I searched the documentation and found the applicable exception:
https://docs.developer.gov.bc.ca/bc-government-organizations-in-github/#bcgov-c

**bcgov-c**
The bcgov-c organization stores temporary (up to 12 months), private repositories with closed-source code and private documents. Closed-source projects must be moved to the bcgov organization at the end of the 12 months. This repository is private.

....

**_Your product team can only have a permanent, private repository in bcgov-c if it is a GitOps repository with ArgoCD manifests._** We strongly discourage creating permanent, private repositories in this organization.
